microservices/rosettafold/microservice/api_models.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from enum import Enum
+from typing import List
+
+import pydantic
+from pydantic import Field
+
+from microservice.mixins import BaseModelMixin
+
+
+@pydantic.dataclasses.dataclass
+class RunRosettaFoldRequest(BaseModelMixin):
+    fasta_content: str | None
+    a3m_content: str | None
+
```

Comment:
you need to add `job_id: str = None` for docking to track the progress of jobs running (will be used for other services as well later). See example in `esmfold` microservice